### PR TITLE
CSS Styling

### PR DESCRIPTION
**Pull request by KainyRyu**
Screenshot 2020-02-28 at 00 16 11
Mobile styling changes

----------------------------------------------------

Screenshot 2020-02-28 at 00 13 56
Screenshot 2020-02-28 at 00 13 48
Screenshot 2020-02-28 at 00 14 06

Seems quite blank, and still needs to improve a lot. 
Also we may need to add a icon or something.
but for now, not too bad i think?

### CODE REVIEW

File: src/frontend/components/Home/Home.js [hunk 1]
```diff
@@ -18,8 +18,9 @@ export default class Home extends React.Component {
         <main>
           <div className="search-background">
             <div className="search-title">
-              <h2><em>"Ready for</em> </h2>
-              <h2><em>Pair Programming"</em></h2>
+              <h1>Start</h1>
+              <p>Find a user</p>
+              <p>Request a pair Programming</p>
```
Comment (AnithaDharaneedharan) [inline src/frontend/components/Home/Home.js]: Can we change the slogan to "Let's Pair Program ! Find your coding partner "

Comment (KainyRyu) [reply src/frontend/components/Home/Home.js]: Won't a bit former sentences suit to the theme??

File: src/frontend/App.css [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
+}
+
+@media(min-width: 600px) {
+  .App{
+    width: 80%;
+    margin: 0 auto;
+    padding: 0;
+  }
+
+  a :visited {
```
Comment (DTFsquad) [inline src/frontend/App.css]: Why do we need the same target element with the same value as width below 600px?
We are repeating what we have on line 24 and 25.

File: src/frontend/App.js [hunk 3]
```diff
@@ -6,6 +6,15 @@ import Home from "./components/Home/Home";
 import Header from "./components/Header/Header";
 import "./App.css";
 
+function NotFound() {
+  return(
+    <>
+      <p>
+      &#47;&#47;page not found
```
Comment (DTFsquad) [inline src/frontend/App.js]: `&#47` What is the reason for the hex period?

File: src/frontend/components/Header/Header.css [hunk 4]
```diff
@@ -1,80 +1,62 @@
...
-      padding: 30px;
-    }
-    
-    .Search-input {
-      width: 300px;
-      height: 50px;
-      margin-top: 10px;
-    }
-  
+  .signing {
```
Comment (DTFsquad) [inline src/frontend/components/Header/Header.css]: I thought that we changed this class name in `sign-in` as per the convention.
Is this a remain that was not changed or this class is out of the convention for naming?

File: src/frontend/components/Header/Header.js [hunk 5]
```diff
@@ -8,24 +8,19 @@ export default function Header() {
   return (
     <div className="Header">
       <div className="navbar">
-        <div className="navbar-logo">
-          <Link to="/">
-            <img className="logo-img" src={logo} alt="logo" />
-          </Link>
-        </div>
-        <div className="navbar-join">
+        <Link to="/">
```
Comment (DTFsquad) [inline src/frontend/components/Header/Header.js]: I see all across some inconsistency:
We name `className="Header"` and `className="Header-bottom"` but on all the others we use lower case.
Any reason behind it?

File: src/frontend/components/Profile/Profile.js [hunk 6]
```diff
@@ -21,31 +21,37 @@ export default class Profile extends React.Component {
...
-      <div>
+      <div className="profile">
         <div className="user-wrapper">
           <img
             className="profile-image"
             src={this.state.userId.image}
             alt={this.state.userId.name}
           />
-          <div className="user-list user-name">{this.state.userId.name}</div>
+          <div className="profile-detail">
```
Comment (DTFsquad) [inline src/frontend/components/Profile/Profile.js]: Can we relook at the indentation?

File: src/frontend/components/SearchBar/SearchBar.css [hunk 7]
```diff
@@ -1,24 +1,45 @@
...
-    margin: 5px auto;
-    
-
+  width: 100%;
+  height: 70px;
+  margin: 1.5px;
+  background-color: #383838;
+  padding-left: 20px;
+  font-size: 20px;
+  color: #cccccc;
```
Comment (DTFsquad) [inline src/frontend/components/SearchBar/SearchBar.css]: We can use the shorthand `color: #ccc;` here

File: src/frontend/components/getUsers.js [hunk 8]
```diff
@@ -62,10 +64,20 @@ const users = [
...
       availability: ["2014-04-11"]
       // availability: ["2014-04-29 14:00:00 UTC" , "2014-05-1 17:00:00 UTC","2014-06-21 20:30:00 UTC"]
-    }
+    },
+    {
+      id: '7',
+      name: "Ricardo",
+      language: "JavaScript, Python, Java",
+      location: "Leytonstone",
+      email: "selchuk.karakus@gmail.com",
```
Comment (DTFsquad) [inline src/frontend/components/getUsers.js]: Chuck e-mail on ricardo profile

File: src/frontend/components/SearchBar/SearchBar.css (hunk 7 above)
Comment (KainyRyu) [reply src/frontend/components/SearchBar/SearchBar.css]: It changes automatically

File: src/frontend/components/getUsers.js (hunk 8 above)
Comment (KainyRyu) [reply src/frontend/components/getUsers.js]: That's dummy data

File: src/frontend/components/Header/Header.js (hunk 5 above)
Comment (KainyRyu) [reply src/frontend/components/Header/Header.js]: `className="Header"` represents the component

**Pull request merged**